src/Feeds.php

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * Pinterest for WooCommerce Feeds related helper methods
+ *
+ * @package     Pinterest_For_WooCommerce/Classes/
+ * @version     1.0.0
+ */
+
+namespace Automattic\WooCommerce\Pinterest;
+
+if ( ! defined( 'ABSPATH' ) ) {
```

Comment:
Let's import our classes here, then we can use them without any prefixes in the code within this file.

```suggestion

use Automattic\WooCommerce\Pinterest\API\Base;
use Exception;

if ( ! defined( 'ABSPATH' ) ) {
```